src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -185,6 +196,58 @@ protected int doWork() {
                     " already exists.  Delete this file and try again, or specify a different output file.");
         }
 
+        // map for aliases mapping a contig to its' aliases
+        final Map<String, Set<String>> contig2aliases = new HashMap<>();
+        
+        // fill the alias map
+        if(this.ALT_NAMES != null) {
+            try {
```

Comment:
This is a lot of code to insert directly into a method. If the output of this block is the `Map<>` above, it shouldn't be hard to move it into a method that returns the `Map`.